CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ aws-parallelcluster-node CHANGELOG
 
 This file is used to list changes made in each version of the aws-parallelcluster-node package.
 
+2.2.0
+-----
+- Prevent jobwatcher crashes by using fallback values
+- Improve nodewatcher logging
+- sqswatcher: add number of slots to the log of torque scheduler
```

Comment:
We could group by module/process, e.g. sqswatcher changes, nodewatcher changes etc
Or we should use the same convention for all the changelog items: E.g. `<processname>: <change>`